doc/test_description.template.rst

```diff
@@ -73,9 +73,13 @@ product versions.
 
 * **OpenDDS**:
 
+  * The commands used to build OpenDDS and the test can be found [here](https://github.com/OpenDDS/OpenDDS/blob/master/.github/workflows/dds-rtps.yml). None of the options used affect interoperability. See the [OpenDDS Developer's Guide](https://opendds.readthedocs.io/en/latest-release/devguide/introduction.html#building-and-configuring-for-interoperability) for additional information about interoperability.
+  * OpenDDS specific configuration for the test can be found in [shape_configurator_opendds.h](https://github.com/omg-dds/dds-rtps/blob/master/srcCxx/shape_configurator_opendds.h). A brief description is:
+    * Use RTPS as the default transport
+    * Use RTPS as the default discovery mechanism
+    * Disable XTypes Support
```

Comment:
I'm not sure if this nested list will render correctly in HTML -- it didn't look right in GitHub's preview but maybe that's just the preview.

Reply:
I think it is correct https://sublime-and-sphinx-guide.readthedocs.io/en/latest/lists.html#unordered-list-inside-unordered-list

Reply:
I thought there were cases where the blank lines mattered.  This may not be one of them.